Hearthstone Deck Tracker/Config.cs

```diff
@@ -1221,7 +1221,13 @@ public void CheckConfigWarnings()
 			}
 		}
 
-		public static void Save() => XmlManager<Config>.Save(Instance.ConfigPath, Instance);
+		public event Action? OnConfigSaved;
+
+		public static void Save()
+		{
+			XmlManager<Config>.Save(Instance.ConfigPath, Instance);
+			Instance.OnConfigSaved?.Invoke();
```

Comment:
There is a [ConfigWrapper](https://github.com/HearthSim/Hearthstone-Deck-Tracker/blob/master/Hearthstone%20Deck%20Tracker/Utility/ConfigWrapper.cs) you can bind the UI to instead of firing on any config change.